src/Command/ThanksCommand.php

```diff
@@ -24,13 +24,23 @@
  */
 class ThanksCommand extends BaseCommand
 {
-    // This is a list of projects that would like to get a star on their main
-    // community repository whenever you use any of their other repositories.
+    // This is a list of projects that should get a star on their main repository
+    // (when there is one) whenever you use any of their other repositories.
+    // When a project's main repo is also a dependency of their other repos (like amphp/amp),
+    // there is no need to list it here, as starring will transitively happen anyway.
     private static $mainRepositories = [
         'api-platform' => [
             'name' => 'api-platform/api-platform',
             'url' => 'https://github.com/api-platform/api-platform',
         ],
+        'cakephp' => [
+            'name' => 'cakephp/cakephp',
+            'url' => 'https://github.com/cakephp/cakephp',
+        ],
+        'doctrine' => [
```

Comment:
I'm not sure about this one. Doctrine packages are not subparts of the `doctrine/doctrine2`. they are separate Doctrine projects (and so, if you use anything using the ORM, it is already thanked)

Reply:
it has been removed since this PR, for the reason you listed :)